packages/renderer/src/lib/objects/KubernetesColumnName.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+import type { KubernetesNamespacedObjectUI, KubernetesObjectUI } from './KubernetesObjectUI';
+
+interface Props {
+  object: KubernetesObjectUI;
+}
+
+let { object }: Props = $props();
+
+function isNamespaced(object: KubernetesObjectUI): object is KubernetesNamespacedObjectUI {
+  return 'namespace' in object;
+}
```

Comment:
nice 👍

Reply:
It was suggested to move the columns under /kube, and we already have a pattern of creating (e.g.) /node/node-utils.ts so I created a kube-utils.ts and moved isNamespaced() there.